lib/impl/testing.dart

```diff
@@ -45,7 +47,9 @@ class TestManager implements Disposable {
     path = _findAssociatedTest(project, path);
 
     if (!allowWithoutTestName) {
-      if (!path.endsWith('_test.dart')) return false;
+      if (!path.endsWith('_test.dart') && !path.endsWith('${_sep}test${_sep}all.dart')) {
```

Comment:
To support `all.dart` in child `test` directories rather than just the top level `test` directory...
```
(!path.contains('${_sep}test${_sep}') || !path.endsWith(${_sep}all.dart'))